### ISSUE TITLE
licenseHeaderFile shows warning when using Kotlin DSL

### ISSUE THREAD

**Issue by Sathawale27** [diffplug/spotless#505]
**Problem:** Shows warning when tried to set the licenseHeaderFile for all subprojects using Kotlin DSL.

**Android Gradle Version:** 4.0.0-alpha07
**Spotless Version:** 3.26.1
**Spotless configuration block:**
```
plugins {
    id("com.diffplug.gradle.spotless") version ("3.26.1")
}

subprojects {
    plugins.apply("com.diffplug.gradle.spotless")

    spotless {
        kotlin {
            target("**/*.kt")
            licenseHeaderFile(file("${project.rootDir}/spotless/copyright.kt"))
        }
    }
}
```
![spotless](https://user-images.githubusercontent.com/7881593/71660036-c2bce300-2d6f-11ea-8ac7-9f059d1f0288.PNG)

**Comment by nedtwigg** [comment]
Thanks for reporting! The fix is trivial, and will be in the next release.

**Comment by JLLeitschuh** [comment]
You know, I've been seeing that warning for almost 2 years now and never reported it. I really should have, sorry about that.

**Comment by nedtwigg** [comment]
Fixed in `3.27.1`
